src/controlCenter/widgets/backlight/backlight.vala

```diff
@@ -0,0 +1,64 @@
+using GLib;
+
+namespace SwayNotificationCenter.Widgets {
+    public class Backlight : BaseWidget {
+        public override string widget_name {
+            get {
+                return "backlight";
+            }
+        }
+
+        BacklightUtil client;
+
+        Gtk.Label label_widget = new Gtk.Label (null);
+        Gtk.Scale slider = new Gtk.Scale.with_range (Gtk.Orientation.HORIZONTAL, 0, 100, 1);
+
+        public Backlight (string suffix, SwayncDaemon swaync_daemon, NotiDaemon noti_daemon) {
+            base (suffix, swaync_daemon, noti_daemon);
+
+            Json.Object ? config = get_config (this);
+            if (config != null) {
+                string ? label = get_prop<string> (config, "label");
+                label_widget.set_label (label ?? "Brightness");
+                string ? device = get_prop<string> (config, "device");
+                string ? subsystem = get_prop<string> (config, "subsystem");
+                if (subsystem != "backlight" && subsystem != "leds") {
+                    info ("Invalid subsystem for device %s. Use 'backlight' or 'leds'. Using default: 'backlight'", device);
```

Comment:
Convert this into a switch statement with the checks in line 33-34 included and this check into `default`.

```vala
switch (subsystem) {
    default:
    case "backlight":
        info ("Invalid subsystem for device %s. " +
              "Use 'backlight' or 'leds'. Using default: 'backlight'",
              device);
        subsystem = "backlight";
        slider.set_range (min, 100);
        break;
    case "leds":
        slider.set_range (min, this.client.get_max_value ());
        break;
}
```